Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -123,6 +125,7 @@ final class MethodModel: Model {
     init(name: String,
          typeName: String,
          kind: MethodKind,
+         encloser: String?,
```

Comment:
No need to pass in encloser here; closuremodel is created lazily during rendering (see render(with:) function in this file), where the encloser is passed in from the mock class. You can change the 'handler' var to a function with an encloser parameter.